### ISSUE TITLE
mtx-select multi-select chips wrap poorly

### ISSUE THREAD

**Issue by llowrey** [ng-matero/extensions#211]
You can see, below, that the the chips are being cut off and there should be some white space between the top and bottom chips.

![image](https://user-images.githubusercontent.com/17756185/235545990-ab64f240-3c67-40da-ba22-8e29a6693223.png)

While my test case is unreasonably narrow, our actual use case tends to have enough items selected to cause the chips to wrap.

**Comment by llowrey** [comment]
I found [`margin: -4px 0;`](https://github.com/ng-matero/extensions/blob/master/projects/extensions/select/select.scss#L78) in `extensions/select/select.scss`.

That makes sense for single select but doesn't work for multi-select. Take that out and it renders nicely.

To get vertical spacing, I tweaked it with `margin-bottom: 3px` for `.ng-value` (in addition to the 4px of right margin) and added `margin-bottom: -3px` to the `.ng-value-container`.

Hope that helps.
